index.d.ts

```diff
@@ -12,6 +12,14 @@ Promise.race([]);
 
 pRace([]);
 //=> [RangeError: Expected the input to contain at least one item]
+
+pRace(signal => [
+	fetch('/api', {signal}),
+	setTimeout(10, {signal}),
+]);
+//=> Remaining promises other than first one will be aborted.
 ```
 */
```

Comment:
The comment is now attached to the wrong thing.

Reply:
What do you mean the comment is attached to the wrong thing? 👀 
The comment points the above `pRace` function call will abort the other promises other than the fastest one.

Reply:
It's pretty clear if you look at the file. The comment is about the `pRace` function, but it's now attached to the `IterableOfPromiseLike` type.

Reply:
Oh, I got this 😓, thanks for clarifying